README.md

```diff
@@ -171,6 +170,44 @@ var userInfo = new RaygunIdentifierMessage("12345")
 
 Log.ForContext("CustomUserInfoProperty", userInfo, true).Error(new Exception("random error"), "other information");
 ```
+
+### onBeforeSend
+`type: Action<OnBeforeSendParameters>`
+
+`default: null`
+
+The field allows users to manipulate the data to be sent to Raygun.
+This is a function that is executed by the raygun client's `CustomGroupingKey` event, and happens at the end of the `OnCustomGroupingKey` event callback.
+By default it is `null`, so you don't need to set it in the constructor.
+If the field is `null`, nothing happens; if an `Action<OnBeforeSendParameters>` is passed, it is executed at the described time.
```

Comment:
I'd move explanation of the null default to the sentence above.

Reply:
Instead of "at the described time" I'd describe that here. Which is that the action gets called just before the crash report payload gets serialized and sent to Raygun.